note-service/src/main/java/ma/enset/noteservice/service/NoteElementService.java

```diff
@@ -0,0 +1,20 @@
+package ma.enset.noteservice.service;
+
+import ma.enset.noteservice.exception.ElementAlreadyExistsException;
+import ma.enset.noteservice.exception.ElementNotFoundException;
+import ma.enset.noteservice.exception.InternalErrorException;
+import ma.enset.noteservice.model.NoteElement;
+import ma.enset.noteservice.model.NoteModule;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+import java.util.List;
+
+public interface NoteElementService {
```

Comment:
Must have updateMany

Reply:
Why do I need to update many note at the same time ?

Reply:
From service to service, when updating multiple objects (notes), we can do it the way you see it, (for every object we send an update request), but it is very slow and inefficient. what we can do is update all objects with only one call. we simply send all objects we want to update in the body request and update them all. (checkout utilisateur-service or element-service)